tests/pools/test_pool_rpc.py

```diff
@@ -38,6 +39,7 @@
 # TODO: Compare deducted fees in all tests against reported total_fee
 log = logging.getLogger(__name__)
 FEE_AMOUNT = 2000000000000
+WAIT_SECS = 200  # A high value for WAIT_SECS is useful when paused in the debugger
```

Comment:
I agree, of course. This is an alternative to having:
```
await time_out_assert(10, ...
await time_out_assert(5, ...
```

One thing to note: this is the maximum time it will wait. If the function returns true earlier, `time_out_assert` will return immediately.

I too look forward to when we don' have these anymore, but that's a ways off.
Also, much better than `sleep`